src/tests/integration/test-data/show/SetUpDeployTest.test_extcodesize().expected

```diff
@@ -5,86 +5,79 @@
 │   callDepth: 0
 │   statusCode: STATUSCODE:StatusCode
 │
-│  (1000 steps)
+│  (713 steps)
 ├─ 3
-│   k: #accessStorage 491460923342184218035706888008750043977755113263 0 ~> 0 ~> #deduc ...
-│   pc: 79
-│   callDepth: 1
-│   statusCode: STATUSCODE:StatusCode
-│
-│  (70 steps)
-├─ 4
 │   k: #end EVMC_SUCCESS ~> #pc [ RETURN ] ~> #execute ~> #codeDeposit 4914609233421842 ...
 │   pc: 145
 │   callDepth: 1
 │   statusCode: STATUSCODE:StatusCode
 │
 │  (1 step)
-├─ 5
```

Comment:
Similarly, this test is not exercised on the legacy backend, so the show output was actually not being tested against before. Now the output is actually being generated by the booster.

Reply:
I've tested that this one actually produces the same output on booster and legacy, and that the update to the test output is correct.